skype/skype-ps/skype/New-CsHuntGroup.md

```diff
@@ -197,16 +196,16 @@ Applicable: Skype for Business Online
 
 Required: False
 Position: Named
-Default value: None
+Default value: DisconnectWithBusy
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -LineUri
-PARAMVALUE: Uri
+### -OverflowActionTarget
+The OverflowActionTarget parameter represents the target of the overflow action. For example, if the OverFlowAction is set to Forward, this parameter might be set to a SIP address or a PSTN phone number. This parameter is optional unless the OverflowAction is set to Forward, which requires a SIP address.
```

Comment:
We should not mention target can be PSTN number

Reply:
Rephrase: "The OverflowActionTarget parameter represents the target of the overflow action. If the OverFlowAction is set to Forward, this parameter must be set to a SIP address. Otherwise, this parameter is optional."

Reply:
Fixed